docs/lab/overview/testing-modalities.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Testing Modalities"
+---
+
+By integrating with our API, you will have access to all testing modalities.
+
+Three types are currently supported:
+
+- At-home test kit.
+- At-home phlebotomy visit.
+- Walk-in test visit.
+
+## At-home test kits
+
+This involves sending a kit to a patients home that allows the patient to collect their
```

Comment:
This involves sending a kit to a patient's home. They will then be able to collect their sample specimen and send it back to our lab networks via mail.